components/LandingPage.js

```diff
@@ -1,22 +1,26 @@
 import React from "react";
-import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { View, Text, StyleSheet, TouchableOpacity, ImageBackground } from "react-native";
 import { Header } from 'react-native-elements';
 import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
     <View style={styles.container}>
       <Header
-        containerStyle={{ backgroundColor: 'black', justifyContent: 'space-around', paddingBottom: 10}}
+        containerStyle={{ backgroundColor: '#483d8b', justifyContent: 'space-around', paddingBottom: 10}}
         centerComponent={{ text: 'Never Alone', style: { color: '#fff', fontSize: normalize(30), fontWeight: "bold" } }}
       />
       <View style={styles.box}>
         <Text style={styles.text}>Choose Your Role:</Text>
-        <TouchableOpacity style={styles.careTakerButton} onPress={() => navigation.navigate('Caretaker')} >
-          <Text style={styles.buttonText}>Caretaker</Text>
-        </TouchableOpacity>
+          <TouchableOpacity style={styles.careTakerButton} onPress={() => navigation.navigate('Caretaker')} >
+          <ImageBackground source={require('../images/caretaker.jpg')} style={styles.backgroundImage} >
+            <Text style={styles.buttonText}>Caretaker</Text>
+            < /ImageBackground>
```

Comment:
ImageBackground is pretty cool. It's weird to see a 'require' in the source. I feel like if we had more time we could make this entire TouchableOpacity into it's own component, as I know we have one below. I definitely don't want to deal with that now though, lol.